plugin/evm/atomic/tx_heap.go

```diff
@@ -1,7 +1,7 @@
-// (c) 2020-2021, Ava Labs, Inc. All rights reserved.
+// (c) 2020-2025, Ava Labs, Inc. All rights reserved.
```

Comment:
Should this be updated on other files? 🤔 
I'm looking into copyright thingies in https://github.com/ava-labs/libevm/pull/100 because right now it's a bit of headache to keep track of those years 😕